src/components/Sidebar/Subjects.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import Divider from '@material-ui/core/Divider';
+
+const styles = theme => ({
+  root: {
+    width: '100%',
+    maxWidth: '360px',
+    backgroundColor: theme.palette.background.paper,
+  },
+});
+
+function ListDividers(props) {
```

Comment:
Uhhh.... what?

Reply:
I was going to start building the presentational component, but I'll remove this file for now from this PR